### DISCUSSION TITLE
Can you add new tags in the run_tags table?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#18278] [Q&A]
Hi we have an issue running these kind of queries, our table is 32 GB in size and there are no correct indexes applied, causing a table scan and performance degradation.

version 1.4.0

Is this a known issue?

```UPDATE run_tags SET value='dagster-producer:prod-v2.0.5' WHERE run_tags.run_id = 'df4fb081-ed42-48a5-8b6e-12aeef6db479' AND run_tags.key = 'dagster/image'```

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1701090837490289?thread_ts=1701090837.490289&cid=C01U954MEER).*

**Comment by jamiedemaria** [comment]
Dupe of https://github.com/dagster-io/dagster/issues/18269
